physical/zookeeper.go

```diff
@@ -52,16 +53,63 @@ func newZookeeperBackend(conf map[string]string) (Backend, error) {
 		machines = "localhost:2181"
 	}
 
+    // zNode owner and schema.
+    var owner string
+    var schema string
+    var schemaAndOwner string
+    schemaAndOwner, ok = conf["znode_owner"]
+    if !ok {
+        owner = "anyone"
+        schema = "world"
+    } else {
+        parsedSchemaAndOwner := strings.SplitN(schemaAndOwner, ":", 2)
+        if !(len(parsedSchemaAndOwner)==2) {
```

Comment:
Minor stylistic comment (here and below), for consistency with the rest of the Vault codebase: please use
```
if len(parsedSchemaAndOwner) != 2 {
```